src/Relation/Binary/Construct/Closure/Transitive.agda

```diff
@@ -70,6 +70,13 @@ module _ (_∼_ : Rel A ℓ) where
   transitive : Transitive _∼⁺_
   transitive = _++_
 
+  transitive⁻ : Transitive _∼_ → _∼⁺_ ⇒ _∼_
+  transitive⁻ trans = ∼⁺⊆∼
+    where
+    ∼⁺⊆∼ : _∼⁺_ ⇒ _∼_
+    ∼⁺⊆∼ [ x∼y ]      = x∼y
+    ∼⁺⊆∼ (x∼y ∷ x∼⁺y) = trans x∼y (∼⁺⊆∼ x∼⁺y)
```

Comment:
Missing CHANGELOG entry as well?

Reply:
Re: overkill
Perhaps you're right. But I think sometimes we don't pay enough attention to observing when certain lemmas really are 'arrow/preorder-theoretic' for structures other than the usual one of implication... so there's a balance to strike between 'minimality' and 'redundant recoding' in my mind as regards how certain things get phrased. I'm not clear in my own mind how much we want to strip down stdlib entries, or have them as exemplary 'staged' versions of how to lay out proofs (for newcomers to follow, and for our students, hopefully, to emulate...)?

In this instance, I think I had had a more articulated version of `_∼⁺_ ⇒ _∼_` and `_∼_ ⇒ _∼⁺_` in mind in order to set up instances of `WellFounded.Subrelation` until I realised that I didn't in fact need them. And this may be a fossil record of that earlier line of thinking.

Reply:
Local recursive definitions vs. (for me!) harder-to-read inlined recursions... a `style-guide` question?
I've gone with your suggestion, but I think it's worth revisiting another time.